Lib/fontmake/font_project.py

```diff
@@ -541,6 +541,18 @@ def save_otfs(
                 if "GSUB" in gsub_src:
                     font["GSUB"] = gsub_src["GSUB"]
 
+
+            # Read autohinting parameters from autohint variable
+            if autohint is not None:
+                autohint_thisfont = autohint
+            # otherwise read autohinting parameters from ufo lib if present
+            elif AUTOHINTING_PARAMETERS in ufo.lib and autohint is None:
+                autohint_thisfont = ufo.lib[AUTOHINTING_PARAMETERS]
+            else:
+                autohint_thisfont = None
+
+            do_autohint = ttf and autohint_thisfont is not None
```

Comment:
you could probably simplify this whole logic like so

```python
autohint_thisfont = (
    ttf and (
        autohint
        or ufo.lib.get(AUTOHINTING_PARAMETERS)
    )
)

if autohint_thisfont:
    ...
```

Reply:
Not really, because in your suggestion `autohint_thisfont` is a boolean, while in my code it contains the autohinting parameters, which I would then still need in a separate variable. My code is analog the old code which also has two separate variables, except that `autohint_thisfont` is now font-specific

Reply:
it's not only a boolean, it can be True, False, None or some string argument.. so technically a `Union[bool, str, NoneType]`. Try it your REPL.

Reply:
I renamed the variable to `autohint_parameters` to be clear

Reply:
Oh. I didn't know that adding a string to a boolean becomes the string or `False`. Okay.

Reply:
https://realpython.com/python-and-operator/#using-pythons-and-operator-with-common-objects

Reply:
Ok, done. Please sign off on this file before I will attempt a test.

Reply:
yes, it's looking ok thanks